src/ripple/app/misc/impl/TxQ.cpp

```diff
@@ -1145,6 +1147,38 @@ TxQ::getMetrics(Config const& config, OpenView const& view,
     return result;
 }
 
+auto
+TxQ::getAccountTxs(AccountID const& account, Config const& config,
+    ReadView const& view) const
+        -> boost::optional<std::map<TxSeq, AccountTxDetails>>
+{
+    auto const allowEscalation =
+        (view.rules().enabled(featureFeeEscalation,
+            config.features));
+    if (!allowEscalation)
+        return boost::none;
+
+    std::map<TxSeq, AccountTxDetails> result;
```

Comment:
My intention was to have the declaration / allocation outside of the lock to save that tiny bit of time under lock. But now that you point it out, it's probably not worth the cost of allocating in the more likely case that there's nothing in the queue for that account, so I'll move it.